testsuite/features/step_definitions/common_steps.rb

```diff
@@ -643,6 +643,26 @@
   $server.run("salt-key -y --accept=pxeboot.example.org")
 end
 
+When(/^I bootstrap traditional client "([^"]*)" using bootstrap script with activation key "([^"]*)" from the proxy$/) do |host, key|
+  # Preparation of bootstrap script for traditional client
+  $proxy.run('mgr-bootstrap --traditional')
+  $proxy.run('sed -i s\'/^exit 1//\' /srv/www/htdocs/pub/bootstrap/bootstrap.sh')
+  $proxy.run("sed -i '/^ACTIVATION_KEYS=/c\\ACTIVATION_KEYS=#{key}' /srv/www/htdocs/pub/bootstrap/bootstrap.sh")
+  $proxy.run('chmod 644 /srv/www/htdocs/pub/RHN-ORG-TRUSTED-SSL-CERT')
+  $proxy.run("sed -i '/^ORG_GPG_KEY=/c\\ORG_GPG_KEY=RHN-ORG-TRUSTED-SSL-CERT' /srv/www/htdocs/pub/bootstrap/bootstrap.sh")
+  output, code = $proxy.run('cat /srv/www/htdocs/pub/bootstrap/bootstrap.sh')
```

Comment:
concatenate all these commands into one string and then run it only once (performance)

Reply:
I agree I will prepare the change.

Reply:
Done.

Reply:
Thank you